CtfPlayback/Metadata/AntlrParser/CtfAntlrMetadataParser.cs

```diff
@@ -95,7 +95,7 @@ private unsafe string GetMetadata(Stream metadataStream)
 
                 // todo:check the endianness based on the magic number
 
-                if (header.Magic == 0x75d11d57) // CTF: Magic Header for binary metadata streams
+                if (header.Magic == 0x75d11d57) // CTF: Magic Header for binary metadata streams // 1976638807
```

Comment:
We probably shouldn't reference header.Magic without checking how many bytes we've actually read - meaning we'll probably need another loop before casting MetadataPacketHeader above.